### ISSUE TITLE
Please rename //clearobjects

### ISSUE THREAD

**Issue by Bastrabun** [Uberi/Minetest-WorldEdit#217]
When we forgot a slash during an attempt to clear all entities from an area, we wiped the a good part of the population of entities on the server, including NPCs and other uniques. Most of them were recovered due to a backup and arduous search and rescue month. It also took us a while to diagnose what was going on and why "WE would suddenly remove all entities" outside of the selected area.

It is simply too easy to forget or even confuse the two commands. Please rename //clearobjects to something unique.

Related request years ago to https://github.com/minetest/minetest/issues/6918

For my reference administration#151

**Comment by sfan5** [comment]
I think adding a prompt to the engine function is the better solution here.
Super simple idea: Force the user to type out the mode `/clearobjects [quick|full]` and don't act upon `/clearobjects`.

**Comment by Bastrabun** [comment]
We renamed the `/clearobjects` command to `/clearobjects_dangerous`, but that's only a solution affecting us after we ran into the problem. This command is not issued often where people would be 100% trained to do the right thing, and even if, I'd rather not have a red button that starts the alarm right next to the red button that blows up a country.

The problem was that `/clearobjects` looked pretty correct for what the user wanted to achieve. Without your, sfan5, quick help on IRC we'd probably not handled the situation right and most likely lost ALL our entities.
